template_engines/backends/odt.py

```diff
@@ -188,3 +188,7 @@ class OdtEngine(ZipAbstractEngine):
     app_dirname = getattr(settings, 'ODT_ENGINE_APP_DIRNAME', 'templates')
     template_class = OdtTemplate
     zip_root_file = 'content.xml'
+
+    def __init__(self, params):
+        params['OPTIONS'] = {'builtins': ['template_engines.templatetags.odt_tags']}
```

Comment:
need to use a dict setdefaults to prevent hard overriding of project configuration

Reply:
or directly use default_builtins in Engine class